src/app/frontend/index.config.ts

```diff
@@ -23,6 +23,10 @@ const SUPPORTED_LANGUAGES: LanguageConfig[] = [
     label: 'German',
     value: 'de',
   },
+  {
+    label: 'فارسی',
```

Comment:
All other languages are listed in English here. Please adjust to make it follow the same pattern.

Reply:
Yes, sorry. I fixed that.